react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class NewsletterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { email: '' };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ email: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    console.log('state:', this.state);
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label htmlFor="email">Email</label>
+        <input type="email" value={this.state.value} id="email" onChange={this.handleChange} />
```

Comment:
your state has an `email` property, use that instead of `this.state.value`

Reply:
Okay, will do